src/inputs/select.scss

```diff
@@ -64,3 +66,67 @@
     }
   }
 }
+
+@mixin iui-multi-select-tag-container {
+  position: absolute;
+  // align wth Select's padding
+  inset: 0 $iui-icons-default + $iui-l 0 $iui-sm;
+  height: 100%;
+  display: flex;
+  align-items: center;
+  gap: $iui-xs;
+  overflow-x: auto;
```

Comment:
```suggestion
  overflow-x: auto;
  overflow-y: hidden;
```
Fixes this issue seen on the small select.
![Screen Shot 2022-05-16 at 6 43 07 PM](https://user-images.githubusercontent.com/849817/168694186-45af057d-c8ec-494d-820c-c4e6716f8f16.png)

Reply:
added `overflow: hidden`